components/QuestionNotificationsToggleExplanationModal.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Modal, ModalHeader, ModalBody, ModalFooter, Button } from 'reactstrap'
+
+const QuestionNotificationsToggleExplanationModal = props => (
+  <Modal isOpen={props.isOpen} toggle={props.toggle}>
+    <ModalHeader>Question notifications</ModalHeader>
+    <ModalBody>
+      <h5>For students</h5>
+      <p>
+        If you enable notificaitons, you&apos;ll get notifications when your
+        question is being answered.
+      </p>
+      <h5>For course staff/admins</h5>
```

Comment:
@nwalters512 Do you mind if I change it to `For on-duty couse staff/admins`, since that is more aligned with our actual logic. You don't get notifications if you don't put yourself on the on-duty staff list.

Reply:
Fine by me!